app/elements/elements.html

```diff
@@ -32,11 +32,6 @@
 <link rel="import" href="../bower_components/paper-input/paper-textarea.html">
 <link rel="import" href="../bower_components/paper-button/paper-button.html">
 
-<!-- Uncomment next block to enable Service Worker Support (2/2) -->
-
-<link rel="import" href="../bower_components/platinum-sw/platinum-sw-cache.html">
```

Comment:
Was not doing anything, and we also don't have it in bower. So if you would run `bower prune`, your index.html would throw 404's